pkgs/tools/admin/gam/default.nix

```diff
@@ -0,0 +1,64 @@
+{
+  fetchFromGitHub,
+  python3,
+  lib,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "gam";
+  version = "6.21";
+  format = "other";
+
+  src = fetchFromGitHub {
+    owner = "GAM-team";
+    repo = "gam";
+    rev = "v${version}";
+    sha256 = "sha256-MNLjdn0YCZ2nasMyuRM72VcGMTbr6jIQyj29+MV1OjQ=";
+  };
+
+  sourceRoot = "source/src";
+
+  patches = [
+    # Also disables update check
+    ./signal_files_as_env_vars.patch
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    dateutil
+    distro
+    filelock
+    google-api-python-client
+    google-auth
+    google-auth-oauthlib
+    passlib
+    pathvalidate
+    setuptools
+  ];
+
+  # Use XDG-ish dirs for configuration. Maybe more than these needed.
+  makeWrapperArgs = [
+      "--set GAMUSERCONFIGDIR '~/.config/gam'"
+      "--set GAMCACHEDIR '~/.cache/gam'"
+      "--set GAMDRIVEDIR '~/'"
+      "--set GAMNOUPDATECHECK '~/'"
+    ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp gam.py $out/bin/gam
+    mkdir -p $out/lib/${python3.libPrefix}/site-packages
+    cp -r gam $out/lib/${python3.libPrefix}/site-packages
+  '';
+
+  checkPhase = ''
+    ${python3.interpreter} -m unittest discover --pattern "*_test.py" --buffer
+  '';
+
+  meta = with lib; {
+    description = "Command line management for Google Workspace";
+    homepage = "https://github.com/GAM-team/GAM/wiki";
+    license = licenses.gpl2;
```

Comment:
This should be either `gpl2Only` or `gpl2Plus` according to [this](https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206).

Reply:
Had the wrong license to begin with.